packages/dataviews/src/add-filter.js

```diff
@@ -117,7 +117,7 @@ export default function AddFilter( { filters, view, onChangeView } ) {
 											return (
 												<DropdownMenuRadioItemCustom
 													key={ element.value }
-													name={ `add-filter-${ filter.field.id }` }
+													name={ `add-filter-${ filter.field }` }
```

Comment:
This rendered `add-filter-undefined` and now it renders `add-filter-author` (for the `author` filter).